### ISSUE TITLE
Unpublish a Package Entirely - com.didimo.sdk.assetfitter

### ISSUE THREAD

**Issue by Hugo-Pereira** [openupm/openupm#3625]
Please read the guideline for [unpublishing a package](https://openupm.com/docs/modifying-upm-package.html#unpublishing-a-package) first.

#### What's the reason to unpublish the package entirely?

This is a preview package, where we were providing tools to facilitate the use of other Didimo products.
Since these tools are very dependent on our clients internal pipelines, it requires tweaking and support from my company. We are moving to a different business strategy, and no longer wish this package to be public.

#### What's the package name?

com.didimo.sdk.assetfitter

#### Have you removed the package meta file and added the package name to the blocked-scopes.yml?

Yes, on this MR https://github.com/openupm/openupm/pull/3624

**Comment by favoyang** [comment]
Closed by #3624

**Comment by favoyang** [comment]
```
$ openupm search com.didimo.sdk.assetfitter
notice No matches found for "com.didimo.sdk.assetfitter"
```

The package has been removed from the registry. The web entry will be removed about 1h later by CI.
